lib/pure/parsecsv.nim

```diff
@@ -159,15 +179,63 @@ proc parseField(my: var CsvParser, a: var string) =
   my.bufpos = pos
 
 proc processedRows*(my: var CsvParser): int =
-  ## returns number of the processed rows
+  ## Returns number of the processed rows.
+  ##
+  ## But even if `readRow <#readRow.CsvParser.int>`_ arrived at EOF then
+  ## processed rows counter is incremented.
+  runnableExamples:
+    import streams
+
+    var strm = newStringStream("One,Two,Three\n1,2,3")
+    var parser: CsvParser
+    parser.open(strm, "tmp.csv")
+
+    doAssert parser.readRow()
+    doAssert 1 == parser.processedRows()
```

Comment:
Change the order of these, please.

```nim
doAssert parser.processedRows() == 1
(...)
doAssert parser.readRow() == false
```

(and in all other examples too)

Reply:
Thx. I did it.

Reply:
Great!

Once you do the remaining unresolved comments, we can merge this.